src/theory/substitutions.h

```diff
@@ -43,33 +43,33 @@ namespace theory {
 class SubstitutionMap {
 
 public:
+ typedef context::CDHashMap<Node, Node> NodeMap;
 
-  typedef context::CDHashMap<Node, Node, NodeHashFunction> NodeMap;
-
-  typedef NodeMap::iterator iterator;
-  typedef NodeMap::const_iterator const_iterator;
+ typedef NodeMap::iterator iterator;
+ typedef NodeMap::const_iterator const_iterator;
 
 private:
+ typedef std::unordered_map<Node, Node> NodeCache;
```

Comment:
Looks like the formatting is wrong because of `private`/`public` not being indented.

Reply:
clang-formatted the whole class, should be fixed now.